exp/landing.md

```diff
@@ -0,0 +1,14 @@
+The explanatory guides in this section introduce you to the concepts used in Anbox Cloud and help you understand how things fit together.
+
+|  |  |
+|--|--|
+| [About Anbox Cloud](https://discourse.ubuntu.com/t/anbox-cloud-overview/17802) | Learn about the differences between Anbox Cloud and the Anbox Cloud Appliance and about the components and architecture of the offering |
+| [About AMS](https://discourse.ubuntu.com/t/about-ams/24321)| Understand the Anbox Management Service (AMS), which handles all aspects of the application and container life cycle |
+| [About applications](https://discourse.ubuntu.com/t/managing-applications/17760)| Learn about application requirements and the application bootstrap process |
+| [About the AAR](https://discourse.ubuntu.com/t/application-registry/17761)| Understand the Anbox Application Registry (AAR) |
```

Comment:
I saw `About AMS` above, maybe for the consistency, we want a  `About AAR`

Reply:
Actually, "About the AMS" would be more correct - I'll update it.